src/compas/datastructures/volmesh/volmesh.py

```diff
@@ -3108,37 +3111,6 @@ def halfface_opposite_halfface(self, halfface):
         nbr = self._plane[w][v][u]
         return None if nbr is None else self._cell[nbr][w][v]
 
-    def halfface_adjacent_halfface(self, halfface, halfedge):
```

Comment:
it can't be that anyone has been using this because the description is nonsensical and the implementation doesn't work. we can leave it in, but i don't know how to fix it because i don't even know what it is supposed to do...